tests/testthat/test_display_functions.r

```diff
@@ -1,4 +1,4 @@
-context('display functions')
+context('general display functions')
 library(repr)
 
 last_data <- NULL
```

Comment:
I'm firmly in the "better RY than complicated stuff", so I'm perfectly happy with this.